rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -1119,10 +1123,10 @@ private void addSvpSpendTransactionInputs(BtcTransaction svpSpendTransaction, Bt
 
     private Coin calculateSvpSpendTxAmount(Federation proposedFederation) {
         int svpSpendTransactionSize = calculatePegoutTxSize(activations, proposedFederation, 2, 1);
-        long backupSizePercentage = (long) 1.2; // just to be sure the amount sent will be enough
+        long svpSpendTransactionBackedUpSize = svpSpendTransactionSize * 12L / 10L; // just to be sure the amount sent will be enough
 
         return feePerKbSupport.getFeePerKb()
-            .multiply(svpSpendTransactionSize * backupSizePercentage)
+            .multiply(svpSpendTransactionBackedUpSize)
```

Comment:
```suggestion
            .multiply(svpSpendTransactionBackedUpSize)
            .multiply(1200) // Add 20% just to be sure the amount sent will be enough
            .divide(1000)
```
Just an idea, maybe we encapsulate the increase here?

Reply:
i found the other way clearer. We would end up having two `.divide(1000)` in a row, kind of confusing. Wdyt?